scripts/new-package.ts

```diff
@@ -0,0 +1,45 @@
+/// <reference types="node" />
+
+import { mkdirSync, writeFileSync } from "fs";
+import * as path from "path";
+
+const newPackageName = process.argv[2];
+if (!newPackageName) {
+    throw new Error("Usage: node scripts/new-package.js new-package-name")
+}
+
+mkdirSync(newPackageName);
+
+write("index.d.ts", `// Type definitions for ${newPackageName} 1.2
```

Comment:
The version should match what's on NPM. (For non-NPM packages, hopefully they have some usable semver.) Could you make a PR for that script?